specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -583,6 +583,86 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/rotateVcenterPassword": {
+      "post": {
+        "operationId": "PrivateClouds_RotateVcenterPassword",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "Rotate the vCenter password",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "202": {
```

Comment:
Is the long running operation return any successful completion status at the end? If the action does not require a response you can use 204 NoContent. Find more here: [https://github.com/Azure/azure-resource-manager-rpc/blob/423d048e58ce13cb8863fb2b2b22b4d6057c5318/v1.0/Addendum.md#call-action-post-asynchronously](https://github.com/Azure/azure-resource-manager-rpc/blob/423d048e58ce13cb8863fb2b2b22b4d6057c5318/v1.0/Addendum.md#call-action-post-asynchronously)